css/components/badge.css

```diff
@@ -0,0 +1,125 @@
+.badge {
+  position: relative;
+}
+
+.badge-icon {
+  border-radius: 50%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  background-color: var(--info-background-color);
+}
+
+/* BADGE FOR XS AVATARS */
+
+.badge-xs {
+  width: 3rem;
+}
+
+.badge-icon-xs {
+  width: 1rem;
+  height: 1rem;
+  top: 0;
+  right: 0;
+  font-size: 0.5rem;
+}
+
+/* BADGE FOR SM AVATARS */
+
+.badge-sm {
+  width: 5rem;
+}
+
+.badge-icon-sm {
+  width: 1.5rem;
+  height: 1.5rem;
+  top: 0;
+  right: 0;
+}
+
+/* BADGE FOR MD AVATARS */
+
+.badge-md {
+  width: 7rem;
+}
+
+.badge-icon-md {
+  width: 2rem;
+  height: 2rem;
+  top: 0;
+  right: 0;
+}
+
+/* BADGE FOR LG AVATARS */
+
+.badge-lg {
+  width: 9rem;
+}
+
+.badge-icon-lg {
+  width: 3rem;
+  height: 3rem;
+  top: 0;
+  right: 0;
+}
+
+/* BADDGES ON ICONS */
+/* ----------------------------------------------- */
+
+.icon-badge {
+  position: relative;
+}
+
+.icon-badge-icon {
+  border-radius: 50%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  background-color: var(--info-background-color);
+}
+
+/* BADGE FOR MD ICONS */
+
+.icon-badge-sm {
+  width: 1.3rem;
+}
+
+.icon-badge-icon-sm {
+  width: 0.7rem;
+  height: 0.7rem;
+  top: -4px;
+  right: -4px;
+  font-size: 10px;
```

Comment:
better to give it in rems. rest all looks good.

Reply:
also, the padding for floating button has cursor pointer but not the button itself. I hope it was not intended.

Reply:
no, didn't know what floating button was so referred to google, that's why, will make everything consistent